templates/home.html

```diff
@@ -89,7 +94,7 @@ <h1 class="main-logo">Kerala<span style="font-weight: normal;">rescue</span></h1
         {% bootstrap_icon "globe" %}
         <span class="text">
             Flood Map<br/>
-           <!-- <span class="ml small notranslate">സഹായം ആവശ്യമായ സ്ഥലങ്ങളും<br/>ദുരിതാശ്വാസ ക്യാമ്പുകളും</span>-->
+           <span class="ml small notranslate">പ്രളയത്തിന്റെ ഭൂപടം</span>
```

Comment:
I think `സഹായം ആവശ്യമായ സ്ഥലങ്ങളും ദുരിതാശ്വാസ ക്യാമ്പുകളും` is the better descriptive than `പ്രളയത്തിന്റെ ഭൂപടം` or you can change it `പ്രെളയബാധിതപ്രദേശങ്ങൾ`

Reply:
Actually that malayalam part was commented out since the microid map does not show camps or requests. The ushahidi map does that now. That's why I did that.

Reply:
Oh Okay, This was just a suggestion.